scripts/ucsc/template-meta-with-python.yaml

```diff
@@ -43,3 +46,9 @@ about:
   home: "http://hgdownload.cse.ucsc.edu/admin/exe/"
   license: "varies; see http://genome.ucsc.edu/license"
   summary: "{summary}"
+  dev_url: "https://github.com/ucscGenomeBrowser/kent"
+  doc_url: "https://github.com/ucscGenomeBrowser/kent/blob/master/README"
+
+extra:
+  additional-platforms:
```

Comment:
Should we add a doi as reference here: https://genome.ucsc.edu/cite.html#ref

Maybe the bigwig, bigbed paper? https://doi.org/10.1093/bioinformatics/btq351

Reply:
Could you please give me the YAML snippet that I should add? Or please push it yourself to this PR!